worlds/tunic/__init__.py

```diff
@@ -74,6 +95,56 @@ def generate_early(self) -> None:
                 self.options.entrance_rando.value = passthrough["entrance_rando"]
                 self.options.shuffle_ladders.value = passthrough["shuffle_ladders"]
 
+    @classmethod
+    def stage_generate_early(cls, multiworld: MultiWorld) -> None:
+        tunc_worlds: Tuple[TunicWorld] = multiworld.get_game_worlds("TUNIC")
+        for tunc in tunc_worlds:
```

Comment:
We should probably rename this
```suggestion
        tunic_worlds: Tuple[TunicWorld] = multiworld.get_game_worlds("TUNIC")
        for tunic in tunic_worlds:
```

Reply:
I feel betrayed, but fair enough